.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>benchflow-spark-tasks-sender</name>
+	<name>Spark-tasks-sender</name>
```

Comment:
In most of the projects in the benchflow repo we are using the benchflow- prefix in the name. This is because when you load the project in an IDE, you get the reference to the BenchFlow framework as part of the project name.